src/node_report_module.cc

```diff
@@ -119,13 +119,13 @@ static void SetSignal(const FunctionCallbackInfo<Value>& info) {
 static void ShouldReportOnFatalError(const FunctionCallbackInfo<Value>& info) {
   Environment* env = Environment::GetCurrent(info);
   info.GetReturnValue().Set(
-      env->isolate_data()->options()->report_on_fatalerror);
+    node::per_process::cli_options->report_on_fatalerror);
```

Comment:
I think you should acquire `node::per_process::cli_options_mutex` in this case, to avoid race conditions when accessing this from multiple threads.

Also, it’s not important, but the previous indentation was the standard one we use for statement continuations (4 spaces).